src/main/java/core/basesyntax/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class AuthenticationException extends Exception {
+
+    public AuthenticationException() {
+    }
+}
```

Comment:
1. Runtime exception is more suitable here as we use it to interrupt application flow and do not plan to handle
2. unnecessary empty line
3. It's a bad practice to create exception without a message
```suggestion
public class AuthenticationException extends RuntimeException {
    public AuthenticationException(String message) {
        super(message);
    }
}
```